app/controllers/reviews_controller.rb

```diff
@@ -12,6 +12,12 @@ def index
 
     @reviews_map[nil] = @reviews.where(semester_id: nil).sort_by{|r| r.course.mnemonic_number}
 
+    average_prof_rating = 0
+    @reviews.each do |r|
+      average_prof_rating += r.overall
+    end
+    @avg_prof_rating = (average_prof_rating/@reviews.count).round(2)
```

Comment:
This seems like something that should be done on our end in the `reviews_controller`.